apps/api/src/app/topics/dtos/add-subscribers.dto.ts

```diff
@@ -1,13 +1,13 @@
 import { ApiProperty } from '@nestjs/swagger';
 import { IsArray, IsDefined } from 'class-validator';
 
-import { SubscriberId } from '../types';
+import { ExternalSubscriberId } from '../types';
```

Comment:
All the confusion came from using the same term both for our internal and the external subscriber ID. Hope the types help to clarify the use when needed.

Reply:
I think it does. Good call.